docs/blog/2024-05-01-active-learning.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Reducing Labeling Costs with Active Learning"
+description: How Groundlight uses active learning to train accurate vision models while saving on data labeling costs.
+slug: active-learning
+authors:
+  - name: Ted Sandler
+    title: Senior Applied Scientist
+    image_url: https://a-us.storyblok.com/f/1015187/1000x1000/efc35da152/sandlert.jpg
+tags: [active learning, uncertainty sampling, deep dive]
+image: ./images/active-learning/social.png
```

Comment:
I'm told that normal humans don't like images that are obviously generated by AI for social posts.  My bad for suggesting such a travesty.